packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -192,6 +193,13 @@ export async function getPermissionsAsync(): Promise<PermissionResponse> {
   return await MediaLibrary.getPermissionsAsync();
 }
 
+export async function presentLimitedLibraryPickerAsync(): Promise<void> {
+  if (MediaLibrary.presentLimitedLibraryPickerAsync) {
+    return await MediaLibrary.presentLimitedLibraryPickerAsync();
+  }
+  return Promise.resolve();
+}
```

Comment:
```suggestion
/**
 * @iOS-only
 * @throws Will throw an error if called on platform that doesn't support this functionality (eg. iOS < 14, Android, etc.).
 // or
 * @returns `true` when the picker is successfully presented and `false` otherwise (eg. iOS < 14, Android, etc.).
 **/
export async function presentLimitedLibraryPickerAsync(): Promise<void> {
  if (MediaLibrary.presentLimitedLibraryPickerAsync) {
    return await MediaLibrary.presentLimitedLibraryPickerAsync();
  }
  return false;
}
```
Isn't using `async` keyword in function signature auto-magically wrap the return value in a `Promise`? 🤔 
I'm not sure if resolving with the same value `void` when nothing happens (on platforms that not support this function) is a good idea 🤔 
What about resolving to `true` if the picker is presented (on iOS 14+) and `false` otherwise (iOS < 14, Android, etc.)?
With the way it's now we have two failing scenario, but one is not reported at all:
- `a)` iOS < 14 - it will throw from the native code
- `b)` no native method (eg. Android) - results as if the modal was presented 🤔 

Other idea it to throw on every platform that is not supporting this functionality 🤔 I'm not sure which approach will be better, but we tend to throw if sth is not supported in other places, so maybe that's the way 🤔 🤷 

cc @cruzach @esamelson

Reply:
I will stick with rejecting the promise if the method is not available.